<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_picture">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>541</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadFromStart">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LoadFromStart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadFromWeb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LoadFromWeb</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_savePicture">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadFromImage">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>500</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LoadFromImage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadFromStop">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LoadFromStop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
